ankitt26/Second-Capstone-project
# Home page and popUp comments

### Added features:
- Added Header
- Added Footer
- Home Page
  - GET items from movies and show API and list them. 
  - POST/GET likes into/from Involvement API
  - Implement the `addLike` button.
  - Count items
  - Test items counter
- Popup comments window
  - Add more details from the movie and show API
  - POST/GET comments into/from Involvement API
  - Add Comment
  - Count Comments
  - Test comments counter
- Documentation
  - Added README file
  - See the [Presentation Video](....)
 
**Try the [Live Demo](...)**
